lib/active_data/model/associations/reflections/embeds_any.rb

```diff
@@ -21,6 +16,21 @@ def self.build(target, generated_methods, name, options = {}, &block)
             super
           end
 
+          private_class_method def self.build_class(superclass)
+            if superclass
+              Class.new(superclass) do
+                include ActiveData::Model
+                include ActiveData::Model::Associations
+                include ActiveData::Model::Lifecycle
+                include ActiveData::Model::Primary
+              end
+            elsif ActiveData.base_concern
+              Class.new { include ActiveData.base_concern }
+            else
+              Class.new(ActiveData.base_class)
+            end
```

Comment:
didn't we discuss that it's poor approach because it will require end users to know internals of underlying gems? I think we should respect all the configuration in any case.
```ruby
if ActiveData.base_concern
  Class.new(superclass || Object) { include ActiveData.base_concern }
elsif superclass do
  Class.new(superclass) do
    include ....
  end
else
  Class.new(ActiveData.base_class)
end
```

P.S. Instead of `private_class_method` you can use
```ruby
class << self
  def pub; end

  private

  def pri; end
end
```

Reply:
@pyromaniac , what do you think about `Class.new(ActiveData.base_class)` ?
```ruby
Class.new(ActiveData.base_class)
```
or
```ruby
Class.new(superclass || ActiveData.base_class) do
    include ...
end
```
From my point of view, the first one is better since we use the same convention for `ActiveData.base_concern`

Reply:
I don't want to fix some hypothetical issues in this PR